src/main/java/io/vertx/rabbitmq/impl/ConsumerHandler.java

```diff
@@ -41,7 +42,7 @@ public ConsumerHandler(Vertx vertx, Channel channel, boolean includeProperties,
 
   @Override
   //TODO: Clean this up, some dup logic here and in basicGet of RabbitMQServiceImpl
-  public void handleDelivery(String consumerTag, Envelope envelope, AMQP.BasicProperties properties, byte[] body) throws IOException {
+  public void handleDelivery(String consumerTag, Envelope envelope, AMQP.BasicProperties properties, byte[] body) {
```

Comment:
is it related to this PR or something different ?

Reply:
something different, i have just reverted the changes